types/sequelize/index.d.ts

```diff
@@ -5105,6 +5105,27 @@ declare namespace sequelize {
     interface Options {
 
         /**
+         * The username which is used to authenticate against the database.
```

Comment:
I'd suggest creating a second separate interfaces for the individual ctor calls as some arguments shouldn't be passed in certain calls as they are already present in another shape or form.